dist/index.js

```diff
@@ -496,8 +497,12 @@ async function run() {
   }
 
   // Pass the logged-in registry URIs to the post action for logout
-  if (registryUriState.length) {
+  if (registryUriState.length && !skipLogout) {
     core.saveState('registries', registryUriState.join());
+  } else {
+    if (skipLogout) {
+      core.debug(`'skip-logout' is TRUE. Will skip logout of ${registryUriState.length} registries.`);
+    }
   }
```

Comment:
To make it clearer, since we only really care that we skip logout if there are logged in registries, can we do something like the following instead?

```
if (registryUriState.length) {
    if (!skipLogout) {
        core.saveState('registries', registryUriState.join());
    }
    core.debug(`'skip-logout' is $skipLogout for ${registryUriState.length} registries.`);
}
```